src/App.js

```diff
@@ -22,7 +27,7 @@ export default function App() {
     { path: '/podcast/:id', Component: DetailsPage },
   ]
   return (
-    <AppContainer>
+    <AppContainer isPlay={play}>
```

Comment:
```suggestion
    <AppContainer isPlaying={play}>
```
just minor and maybe this is not the right word but for me `isPlay` is not intuitive